MAINTAINERS.md

```diff
@@ -0,0 +1,103 @@
+
+# Maintainers
+
+## Maintainer Scopes, GitHub Roles and GitHub Teams
+
+Maintainers are assigned the following scopes in this repository:
+
+| Scope | Definition | GitHub Role | GitHub Team |
+| ----- | ---------- | ----------- | ----------- |
+| Maintainer | The GitHub Maintain role | Maintain   | `<repository> committers`     |
```

Comment:
@davidpoltorak-io I don't follow the purpose of the GitHub Team column. Can you elaborate?

Reply:
This refers to how access has been set up within Github. Those that are part of a specific `team` get given the right to `maintain` the repository. For this repo - all maintainers are part of the `open-enterprise-agent-maintainers` team and have the role of `Maintain`